iris_grib/_save_rules.py

```diff
@@ -634,17 +632,61 @@ def grid_definition_template_30(cube, grib):
                       int(np.round(first_x / _DEFAULT_DEGREES_UNITS)))
     eccodes.codes_set(grib, "LaD", cs.central_lat / _DEFAULT_DEGREES_UNITS)
```

Comment:
Having discussed with @pp-mo, the current code is perfectly correct since this value is always identical to central latitude in the polar stereographic case. What _does_ need improving is preventing users from saving any other value, since they could conceivably create a `Stereographic` instance with an alternative `true_scale_lat` but which would never be saved - need a `TranslationError`.

Reply:
079fcfce4eb10c32c4b5e1cbd61227400e792e7c